e2e/runner/evm.go

```diff
@@ -97,6 +97,24 @@ func (r *E2ERunner) ERC20Deposit(
 	return tx
 }
 
+// ZetaDeposit calls Deposit of Gateway with zeta token
+func (r *E2ERunner) ZetaDeposit(
```

Comment:
```suggestion
func (r *E2ERunner) ZETADeposit(
```

Reply:
I dont mind making this change but we use , Zeta throughout the codebase 
Such as 
LegacySendZetaOnEvm
MintZetaToEVMAccount
etc

imo either is fine as long as we are consistent